### PR TITLE
Update Player's vfx on Boss scene

### PR DESCRIPTION
**Pull request by alvarosoppelsa**
On Boss scene were updated player's vfx for health, chargind and shooting. For all the levels some particles were added for dashing

### CODE REVIEW

**Review by monikamnc — CHANGES_REQUESTED** [review]
Stop until we have new scenes updated, sorry >.

**Review by bernatgb — APPROVED** [review]
Good job! However I would exaggerate more the dash particles.

**Comment by alvarosoppelsa** [comment]
> Good job! However I would exaggerate more the dash particles.

Ok! Like more particles, particles stay longer time, bigger particles?

**Review by monikamnc — APPROVED** [review]
Unblock

**Comment by alvarosoppelsa** [comment]
> Unblock

Sadly it isn't

**Review by Izenz — APPROVED** [review]
Looks good!

File: Gameplay/CutsceneManager.h [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <scripting\Script.h>
```
Comment (Izenz) [inline Gameplay/CutsceneManager.h]:
```suggestion
#include <scripting/Script.h>
```
You should change this btw.

**Pull request merged**